bin/main.rb

```diff
@@ -0,0 +1,5 @@
+puts '****************'
+puts '|              |'
+puts '| Hello world! |'
+puts '|              |'
+puts '****************'
```

Comment:
- **_The `main.rb` should be an [executable file](https://commandercoriander.net/blog/2013/02/16/making-a-ruby-script-executable/): .i.e. `#!/usr/bin/env ruby` is the top line_**